vuepress/docs/next/tutorials/compose/content-types-tutorial.md

```diff
@@ -1,101 +1,70 @@
 # Content Types
 
-A Content Type is used to represent or to model a Content.
-It is characterized by a set of [Attribute Types](./content-attributes.md), each of which defines the value of their properties, called Attributes. A Content is
-an instance or specialization of the Content Type.
+A Content Type represents or models a Content. Alternatively, a Content is an instance or specialization of a Content Type. 
 
-## Create a Content Type
-
-Creating Contents Types is fairly simple. The Content Types are managed
-through the Web CMS, its user interface is made available in the App
-Builder application.
-
--   Go to `Content → Types`
+Content Types are characterized by [Attribute Types](./content-attributes.md). Each Attribute Type consists of one or more properties, known as Attributes. An Attribute is defined by its Attribute Type.
```

Comment:
Each Attribute Type consists of one or more properties, known as Attributes. An Attribute is defined by its Attribute Type. >>these sentences are redundant?

Reply:
reading it carefully and thinking it through i don't think so... the original phrasing implies that the type defines the property, not the other way around, and working through the exercise that's indeed the case